topics/02_Controls/igGrid/00_igGrid_Overview.md

```diff
@@ -41,6 +41,17 @@ Further, the grid also includes support for:
 -   Rich client-side API
 -   ASP.NET MVC wrapper
 
+## Adding igGrid using the Ignite UI CLI
+The easiest way to add a new igGrid to your application is via the Ignite UI CLI. After you have created a new application, you just need to execute the following command and a grid will be added to the project:
+
+```
+ig add grid newGrid
```

Comment:
You have to go with the full set of commands to get to the add. You start with installing the CLI, through a new project and then add.

Reply:
We have the same structure in the editors - so if those are updated we need to update editors' topics too. Actually for adding topics - it's best to unify the sentence so the users don't get confused. We can make "Ignite UI CLI" link to "https://github.com/IgniteUI/igniteui-cli/wiki" and new application part can have link to "https://github.com/IgniteUI/igniteui-cli/wiki/new"

Reply:
I modified the topics to include the full set of commands from installing Ignite UI CLI to building and serving the project, Similar to the way they are listed for the [Angular CLI](https://cli.angular.io/)